apps/nextjs-website/src/_contents/products.ts

```diff
@@ -99,4 +99,24 @@ export const spaceToPrefixMap = pipe(
   RA.uniq(Eq.struct({ spaceId: s.Eq, pathPrefix: s.Eq }))
 );
 
+export const urlRewritesMap: { readonly [url: string]: string } = {
+  'manuale-operativo-dei-servizi': 'manuale-servizi',
+  'kb-enti': 'supporto-agli-enti',
+  'kb-enti-adesione': 'supporto-agli-enti',
+  'kb-enti-servizi': 'supporto-agli-enti',
+  'kb-enti-messaggi': 'supporto-agli-enti',
+  'kb-enti-pagamenti': 'supporto-agli-enti',
+  'kb-enti-accordi': 'supporto-agli-enti',
+  'kb-enti-assistenza': 'supporto-agli-enti',
+  'kit-di-comunicazione-per-gli-enti': 'kit-comunicazione',
+  'io-come-aderire': 'accordi-adesione',
+  'f.a.q.-per-integratori': 'knowledge-base',
+  'modello-di-integrazione-di-piattaforma-notifiche': 'modello-di-integrazione',
+  'manuale-operativo-back-office-pagopa-ente-creditore': 'manuale-bo-ec',
+  'manuale-bo-pagopa-psp': 'manuale-bo-psp',
+  gestionedeglierrori: 'errori',
+  'dizionario-dei-metadata': 'metadata',
+  'manuale-operativo-di-firma-con-io': 'manuale-operativo',
+};
```

Comment:
I would simplify it like this:

```TypeScript
export const urlRewritesMap = {
  'https://docs.pagopa.it/manuale-operativo-dei-servizi': 'manuale-servizi',
  'https://docs.pagopa.it/kb-enti': 'supporto-agli-enti',
  'https://domain.com/i-can/put/here/any/path': 'and-is-going/to/be-replaced/with-this-one',
  ...
};
```

By doing this, the logic in `gitbook-docs` can be simplified using a straightforward replacement. For any given URL, you can simply replace the left side with the corresponding right side. 

# Example
Given the following url `https://docs.pagopa.it/modello-di-integrazione-di-piattaforma-notifiche/v/v2.0/` and the following rewrites map:
```TypeScript
export const urlRewritesMap = {
  'https://docs.pagopa.it/modello-di-integrazione-di-piattaforma-notifiche/v/': '/send/guides/modello-di-integrazione/`
  ...
};
```
The system will produce the following url `/send/guides/modello-di-integrazione/v2.0/`